# Total planning attempts
max_solutions: 10

# Planning group and link names
arm_group_name: "arm"
eef_name: "end_effector"
hand_group_name: "gripper"
hand_frame: "j2s6s200_end_effector"
finger_tip_names: ["j2s6s200_link_finger_tip_1","j2s6s200_link_finger_tip_2"]
#camera_optical_frame : "j2s6s200_wrist_rgbd_depth_optical_frame"

# Cartesian move
move_linear : 0.10
move_angular : 0.30

# Poses
hand_open_pose: "Open"
hand_close_pose: "Close"
arm_home_pose: "Home"

# Scene frames
world_frame: "robot_base_footprint" # world / robot_base_footprint
table_reference_frame: "robot_base_footprint"
object_reference_frame: "robot_base_footprint"
camera_reference_frame: "robot_base_footprint"
surface_link: "table"

# Collision object for picking
object_name: "pick_target"
object_dimensions: [0.25, 0.02] # [height, radius]

# Gripper grasp frame transform [x,y,z,r,p,y]
grasp_frame_transform: [0, 0.0, -0.01, 1.5708, 0, 1.5708]

# Valid distance range when approaching an object for picking
approach_object_min_dist: 0.1
approach_object_max_dist: 0.15

# Valid height range when lifting an object after pick
lift_object_min_dist: 0.01
lift_object_max_dist: 0.1

# Look pose 
look_dist: 0.30

# Place pose [x,y,z,r,p,y]
# place_pose: [-0.03, -2, 0.05, 0, 0, 0]
place_pose: [-0.03, -2, -0.01, 0, 0, 0]
place_surface_offset: 0.0001 # place offset from table

#MoveIt warehouse parameters 
host: "localhost"
port: 33829

#MoveIt constraint
moveit_constraint: "shoulder_elbow" #Motion planning constraint to be used
